context/app/static/js/components/Home/BarChart/BarChart.jsx

```diff
@@ -1,8 +1,12 @@
 import React from 'react';
-import { Placeholder } from './style';
+import { Vega as ReactVega } from 'react-vega';
 
-function BarChart() {
-  return <Placeholder />;
+function BarChart(props) {
+  const { data, spec: partialSpec } = props;
+  const spec = { ...partialSpec, data: { name: 'table' } };
+
+  // eslint-disable-next-line prettier/prettier
```

Comment:
why? If it doesn't work for us, should we tweak the prettier config?

Reply:
Fixed.